Frameworks/CoreFoundation/Preferences.subproj/CFPreferences.c

```diff
@@ -882,11 +882,7 @@ static Boolean synchronizeApplicationDomain(CFTypeRef context, void *domain) {
     CFDataRef data = CFPropertyListCreateData(__CFPreferencesAllocator(), (CFPropertyListRef)domain, kCFPropertyListXMLFormat_v1_0, 0, NULL);
     Boolean success = false;
     if (data) {
-#if DEPLOYMENT_TARGET_WINDOWS
-        success = _CFWriteBytesToFileAsync((CFURLRef)context, CFDataGetBytePtr(data), CFDataGetLength(data));
```

Comment:
If we do this, we would still regress #561, perhaps there is some perf improvement to be gained there?